File: Section1 - Basics/PatternsJava/pattern15.java
void printTriangle(int n) {
    // code here
    for(int i=n; i>0; i--) {
        int k=65;
        for(int j=1; j<=i; j++) {
            System.out.print((char)k);
            k++;
        }
        System.out.print("\n");
    }
}

File: Section1 - Basics/Patterns/pattern15.js
function printTriangle(n) {
	let str = "";
	for (let i = n; i > 0; i--) {
		let k = 65;
		for (let j = 1; j <= i; j++) {
			str = str + String.fromCharCode(k);
			k++;
		}
		str += "\n";
	}
	console.log(str);
}
printTriangle(5);
